presto-prometheus/src/main/java/io/prestosql/plugin/prometheus/PrometheusTimeSeriesValue.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.prometheus;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import java.sql.Timestamp;
+
+@JsonFormat(shape = JsonFormat.Shape.ARRAY)
+public class PrometheusTimeSeriesValue
+{
+    @JsonDeserialize(using = PrometheusTimestampDeserializer.class)
+    public Timestamp timestamp;
+    @JsonProperty
+    public String value;
```

Comment:
make fields private and create proper constructor etc

Reply:
Done in 6ead9ca2